main/seccomp-bpf.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * seccomp example for x86 (32-bit and 64-bit) with BPF macros
+ *
+ * Copyright (c) 2012 The Chromium OS Authors <chromium-os-dev@chromium.org>
+ * Authors:
+ *  Will Drewry <wad@chromium.org>
+ *  Kees Cook <keescook@chromium.org>
+ *
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+#ifndef _SECCOMP_BPF_H_
+#define _SECCOMP_BPF_H_
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE 1
+#endif
+
+#include <config.h>
```

Comment:
https://ci.appveyor.com/project/masatake/ctags/build/1.0.2153/job/3a9rn0kq385ct8ow#L78
"config.h" is not always available. Should be surrounded by `#ifdef HAVE_CONFIG_H`...`#endif`.